spec/i18n_spec.rb

```diff
@@ -17,6 +17,14 @@
   es: /¿|ó/,
 }.freeze
 
+# Regex patterns for commonly misspelled words by locale. Match on word boundaries ignoring case.
+# The current design should be adequate for a small number of words in each language.
+# If we encounter false positives we should come up with a scheme to ignore those cases.
+# Add additional words using the regex union operator '|'.
+COMMONLY_MISSPELLED_WORDS = {
+  en: /\b(cancelled|occured|seperated?)\b/i,
+}
```

Comment:
Minor: Since this is constant, we could freeze it as we're doing above to avoid any unintended changes?

```suggestion
}.freeze
```